ext/opentelemetry-ext-sqlalchemy/README.rst

```diff
@@ -0,0 +1,23 @@
+OpenTelemetry SQLAlchemy Tracing
+================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-ext-sqlalchemy.svg
+   :target: https://pypi.org/project/opentelemetry-ext-sqlalchemy/
+
+This library allows tracing requests made by the SQLAlchemy library.
```

Comment:
Maybe it might be good to call out popular databases that we don't have integrations for that are supported by SQLAlchemy? Or include a link? I get some customers that are confused on which integrations to use for their databases, and we should have a consistent recommendation. I believe we should recommend the specific integrations if they exist (like for mysql), and to try to use SQLAlchemy otherwise (if supported).

Reply:
I added a link to sqlalchemy to the reference section. Regarding recommendations, I wonder where the best place to put something like this is, seems like it could be part of the examples section 🤔

Reply:
Maybe we can just verbally indicate in the README: "For MYSQL use the integration(link), for postgres use the integration(link). For all other databases that support SQLAlchemy, use this..."